src/coreclr/src/jit/lclvars.cpp

```diff
@@ -7390,6 +7392,11 @@ Compiler::fgWalkResult Compiler::lvaStressLclFldCB(GenTree** pTree, fgWalkData*
             tree->ChangeOper(GT_LCL_FLD);
             tree->AsLclFld()->SetLclOffs(padding);
         }
+        else if (oper == GT_LCL_VAR_ADDR)
+        {
+            tree->ChangeOper(GT_LCL_FLD_ADDR);
```

Comment:
There's code above that sets `GTF_GLOB_REF` on `tree`, that should not be done on `GT_LCL_VAR|FLD_ADDR` nodes. I'm not sure if that matters for stress mode but then I'm not sure why this particular stress mode exist to begin with.